lib/generators/solidus_starter_frontend/solidus_starter_frontend_generator.rb

```diff
@@ -5,26 +5,39 @@ class SolidusStarterFrontendGenerator < Rails::Generators::Base
 
   def install
     # Copy directories
-    directory 'app', 'app'
-    directory 'lib/views', 'lib/views'
+    directory 'app', 'app', exclude_pattern: /auth_views/
```

Comment:
I don't understand why we need this part: `exclude_pattern: /auth_views/`. There is no `auth_views` folder in the app directory, isn't it?

Reply:
It's because of this file:

https://github.com/nebulab/solidus_starter_frontend/blob/waiting-for-dev/fix_executable_installation/app/controllers/concerns/solidus_starter_frontend/auth_views.rb

We don't need it, thanks to your suggestion to add the views directly into the application. However, it crashes because of `Engine` not being within the namespace if we copy it.